lib/prompts/checkbox.js

```diff
@@ -67,6 +67,10 @@ Prompt.prototype._run = function (cb) {
   events.normalizedDownKey.takeUntil(validation.success).forEach(this.onDownKey.bind(this));
   events.numberKey.takeUntil(validation.success).forEach(this.onNumberKey.bind(this));
   events.spaceKey.takeUntil(validation.success).forEach(this.onSpaceKey.bind(this));
+  events.allKey.takeUntil(validation.success).forEach(this.onAllKey.bind(this));
+  events.noneKey.takeUntil(validation.success).forEach(this.onNoneKey.bind(this));
+  events.inverseKey.takeUntil(validation.success).forEach(this.onInverseKey.bind(this));
+  events.randomKey.takeUntil(validation.success).forEach(this.onRandomKey.bind(this));
```

Comment:
If we drop random (given your comment below), this line should probably disappear?